src/domain/event/hooks/useEventFieldsData.ts

```diff
@@ -0,0 +1,33 @@
+import { INCLUDE, KEYWORD_SETS } from '../../../constants';
+import {
+  useKeywordSetQuery,
+  useLanguagesQuery,
+} from '../../../generated/graphql';
+import getPathBuilder from '../../../utils/getPathBuilder';
+import { keywordSetPathBuilder } from '../../keywordSet/utils';
+
+const useEventFieldsData = () => {
```

Comment:
This could use some explaining. Not very evident what this does

Reply:
Renamed useEventFieldsData as useEventFieldOptionsData and added comment. I hope it's clearer now. Also started using it on audience section and classification section so added audienceData and topics data to return value